common/changes/@gooddata/sdk-ui-all/anvuong-sd1289_2020-12-13-15-56.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
As the changes are in a component marked as `@internal` the changelog entry is not needed. Please delete this file.